Libraries/D3D12RaytracingFallback/src/FallbackLayerCompilerRepro/FallbackLayerCompilerRepro.vcxproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these kinds of files supposed to be checked in?

Reply:
Apparently yes. [Here's an example.](https://github.com/Microsoft/DirectX-Graphics-Samples/blob/master/Libraries/D3D12RaytracingFallback/src/FallbackLayer.vcxproj.user)

Reply:
Ok